docs/maintainers/best-practice.md

```diff
@@ -0,0 +1,50 @@
+# Maintainer Best Practices
+
+## PR Types
+
+- Feature-work PR: A PR that implements an RFC, which usually involves relatively large set of changes.
+- Regular PR: A bug fix or an enhancement change that are not backed by an RFC.
+
+## Review PRs
+
+- Ask the author to reword the PR title based on guidelines in [Contributing](../../.github/CONTRIBUTING.md).
+- Ask the author to apply `[Feature]` tag to trigger full test builds if it's necessary.
+- Label the PR with `Breaking-Change` and/or `Documentation Needed` as appropriate.
```

Comment:
I would also add "... and add PR classification using other Labels if they are missing".

Reply:
Good point. Will update.